src/structurizr/view/view_set.py

```diff
@@ -75,25 +77,85 @@ class ViewSet(ModelRefMixin, AbstractBase):
     def __init__(
         self,
         *,
-        model: Optional["Model"] = None,
+        model: Optional["Model"],
         # TODO:
         # enterprise_context_views: Iterable[EnterpriseContextView] = (),
         system_landscape_views: Iterable[SystemLandscapeView] = (),
         system_context_views: Iterable[SystemContextView] = (),
         container_views: Iterable[ContainerView] = (),
+        component_views: Iterable[ComponentView] = (),
         configuration: Optional[Configuration] = None,
         **kwargs
     ) -> None:
         """Initialize a view set."""
         super().__init__(**kwargs)
         # self.enterprise_context_views = set(enterprise_context_views)
-        self.system_landscape_views = set(system_landscape_views)
-        self.system_context_views = set(system_context_views)
-        self.container_views = set(container_views)
+        self.system_landscape_views: Iterable[SystemLandscapeView] = set(
```

Comment:
Again, if these annotations are necessary at all.

```suggestion
        self.system_landscape_views: Set[SystemLandscapeView] = set(
```